### ISSUE TITLE
Bug: v7.2 tour points not loading

### ISSUE THREAD

**Issue by wuwinson** [mattermost/focalboard#3663]
### Tour points not loading
1. Go to [bug bash server](https://boards-bugbash.test.mattermost.cloud/focalboard/channels/town-square) and switch over to boards
2. See error - v7.2 tour points don't load when you first open Boards. Tour points only load after finishing the Welcome tour.

## Expected behavior

v7.2 tour points load when you first open Boards v7.2 for the first time

## Edition and Platform

 - Edition: Mattermost Boards (plugin)
 - Version: v7.2 RC3
 - Browser and OS: Chrome on Mac

**Comment by wuwinson** [comment]
cc @sbishel

**Comment by Rajat-Dabade** [comment]
@wuwinson, I was able to see the 7.2 tours, when I log in to the bug-bush server for the first time. I have checked it with @harshilsharma63 as well, and he was also able to see the tour for the first time. 

I suspect that you might have not completed the first tour and kept it as it was and never completed it, which will only resume when you have an onboarding welcome board and complete each step. 

My understanding is:
- 7.2 version tours for the existing users will only take place if they 
	- have completed the first tour before 7.2 was released.
	- skipped the tour from the welcome page before 7.2 was released.
	- if they took the product tour again.  
- For the fresh user, we have a welcome page which will be responsible for skipping the whole tour or taking a full product tour including 7.2 tours as well. 

cc @sbishel
